### ISSUE TITLE
 Failed to download vulnerability database in rootfs scan

### ISSUE THREAD

**Issue by Miladbr** [aquasecurity/trivy#1953]
## Description

Similar to issue #1940 rootfs scan fails at downloading vulnerability database.  

## Output of run with `-debug`:

```
2022-04-06T01:12:07.438+0430    DEBUG   Severities: UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
2022-04-06T01:12:07.440+0430    DEBUG   cache dir:  /root/.cache/trivy
2022-04-06T01:12:07.440+0430    DEBUG   There is no valid metadata file: unable to open a file: open /root/.cache/trivy/db/metadata.json: no such file or directory
2022-04-06T01:12:07.441+0430    INFO    Need to update DB
2022-04-06T01:12:07.441+0430    INFO    DB Repository: 
2022-04-06T01:12:07.441+0430    INFO    Downloading DB...
2022-04-06T01:12:07.442+0430    DEBUG   no metadata file
2022-04-06T01:12:07.442+0430    FATAL   DB error:
    github.com/aquasecurity/trivy/pkg/commands/artifact.runWithTimeout
        /home/runner/work/trivy/trivy/pkg/commands/artifact/run.go:87
  - failed to download vulnerability DB:
    github.com/aquasecurity/trivy/pkg/commands/operation.DownloadDB
        /home/runner/work/trivy/trivy/pkg/commands/operation/operation.go:111
  - OCI artifact error:
    github.com/aquasecurity/trivy/pkg/db.(*Client).Download
        /home/runner/work/trivy/trivy/pkg/db/db.go:151
  - OCI artifact error:
    github.com/aquasecurity/trivy/pkg/db.(*Client).populateOCIArtifact
        /home/runner/work/trivy/trivy/pkg/db/db.go:188
  - repository name error (:2):
    github.com/aquasecurity/trivy/pkg/oci.NewArtifact
        /home/runner/work/trivy/trivy/pkg/oci/artifact.go:50
  - could not parse reference: :2


```

## Output of `trivy -v`:

```
Version: 0.25.2
```

**Comment by DmitriyLewen** [comment]
Hello @Miladbr 
Thanks a lot for your report!

We will fix it soon.

Best Regards, Dmitriy

**Comment by DmitriyLewen** [comment]
@Miladbr fix this bug will be in v0.25.3.

**Comment by d-niu** [comment]
I'm on v0.25.3 and getting this error 

```
FATAL	DB error: failed to download vulnerability DB: OCI artifact error: OCI artifact error: repository name error (:2): could not parse reference: :2
parse error: Invalid numeric literal at line 1, column 14
```

In particular:

```
2022-04-12T19:11:30    INFO    Need to update DB
2022-04-12T19:11:30    INFO    DB Repository: 
2022-04-12T19:11:30    INFO    Downloading DB...
2022-04-12T19:11:30    FATAL   DB error: failed to download vulnerability DB: OCI artifact error: OCI artifact error: repository name error (:2): could not parse reference: :2

```


When running `trivy sbom --artifact-type fs reponame`

**Comment by knqyf263** [comment]
@d-niu Thanks for the report. It was fixed in 0.25.4.
https://github.com/aquasecurity/trivy/releases/tag/v0.25.4
